packages/pluggableWidgets/area-chart-web/src/AreaChart.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from "classnames";
+import { createElement, ReactElement, useCallback } from "react";
+import { AreaChartContainerProps, SeriesType } from "../typings/AreaChartProps";
+import { ChartWidget, ChartWidgetProps } from "@mendix/shared-charts";
+import { getPlotChartDataTransforms, SeriesMapper, usePlotChartDataSeries } from "@mendix/shared-charts/hooks";
+// import { getPlotChartDataTransforms, usePlotChartDataSeries } from "@mendix/shared-charts/hooks";
+
+const areaChartLayoutOptions: ChartWidgetProps["layoutOptions"] = {
+    xaxis: {
+        zeroline: true,
+        fixedrange: true,
+        gridcolor: "#d7d7d7",
+        zerolinecolor: "#d7d7d7"
+    },
+    yaxis: {
+        fixedrange: true,
+        gridcolor: "#d7d7d7",
+        zeroline: true,
+        zerolinecolor: "#d7d7d7"
+    }
+};
+const areaChartConfigOptions: ChartWidgetProps["configOptions"] = {
+    responsive: true
+};
+const areaChartSeriesOptions: ChartWidgetProps["seriesOptions"] = {};
+
+export function AreaChart(props: AreaChartContainerProps): ReactElement {
+    const mapSeries = useCallback<SeriesMapper<SeriesType>>(
+        (line, dataPoints) => ({
+            type: "scatter",
+            fill: "tonexty",
+            fillcolor: line.fillcolor?.value,
+            mode: line.lineStyle === "line" ? "lines" : "lines+markers",
```

Comment:
I wasn't able to find mode for area chart in the documentation, does this even change anything here?
Because I believe this mode values are different from chart to chart.

In BubbleChart it was only markers, so I removed every check with `lines` and `lineStyles`

Reply:
@raffiar, this property only make sense when you have `type="scatter"`, and yes -- it works for Area chart too